tests/chaos/oadp/conftest.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import logging
+
+import pytest
+from ocp_resources.datavolume import DataVolume
+from ocp_resources.utils.resource_constants import ResourceConstants
+
+from utilities.constants import FILE_NAME_FOR_BACKUP, TEXT_TO_TEST, TIMEOUT_3MIN, TIMEOUT_5MIN
+from utilities.infra import ExecCommandOnPod, wait_for_node_status
+from utilities.oadp import VeleroBackup, create_rhel_vm, is_storage_class_support_volume_mode
+from utilities.storage import write_file
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def rhel_vm_with_dv_running(request, chaos_namespace, snapshot_storage_class_name_scope_module):
+    """
+    Create a RHEL VM with a DataVolume, supporting both Block and Filesystem volumes.
+    The volume_mode should be provided in request.param['volume_mode'].
+    """
+    param = request.param
+
+    # Default to Filesystem if not specified
+    requested_mode = param.get("volume_mode", DataVolume.VolumeMode.FILE)
+    if requested_mode.lower() == "block":
+        volume_mode = "Block"
+        # Only check Block support
+        if not is_storage_class_support_volume_mode(snapshot_storage_class_name_scope_module, "Block"):
+            pytest.skip(
+                f"Storage class {snapshot_storage_class_name_scope_module!r} doesn't support volume mode 'Block'"
+            )
+    else:
+        volume_mode = "Filesystem"
```

Comment:
in another thread about marker you say that it can work with both Block and FS filesystems.
so why do you need this then?